Resources/Prototypes/_Echo/Entities/Objects/Fun/centcom_pai.yml

```diff
@@ -0,0 +1,42 @@
+- type: entity
+  parent: PersonalAI
+  id: CentComPersonalAI
+  name: CentCom personal ai device
+  description: Your Central Command pal who might be fun to be with!
+  components:
+  - type: PAI
+    namePrefix: pai-system-name-prefix-centcom
+  - type: PAICentCom
+  - type: Sprite
+    sprite: _Echo/Objects/Fun/centcom_pai.rsi
+    layers:
+    - state: centcom-pai-base
+    - state: centcom-pai-off-overlay
+      shader: unshaded
+      map: [ "screen" ]
+  - type: ToggleableGhostRole
+    roleName: pai-system-role-name-centcom
+    roleDescription: pai-system-role-description-centcom
+    roleRules: ghost-role-information-familiar-rules
+  - type: GenericVisualizer
+    visuals:
+      enum.ToggleableGhostRoleVisuals.Status:
+        screen:
+          Off: { state: centcom-pai-off-overlay }
+          Searching: { state: centcom-pai-searching-overlay }
+          On: { state: centcom-pai-on-overlay }
+
+- type: entity
+  id: ActionPAIElectrocute
+  name: Electrocute
+  description: Electrocute whoever is near you.
+  components:
+  - type: InstantAction
+    checkCanInteract: false
+    checkConsciousness: false
+    itemIconStyle: BigAction
+    priority: 20
+    icon:
+      sprite: _Echo/Interface/Actions/pai-shock.rsi
+      state: pai-shock
+    event: !type:PAIElectrocuteActionEvent {}
```

Comment:
There should be a reasonable cooldown. `InstantAction` has a cooldown setting.

Reply:
InstantAction's cooldown is not a datafield nor is it used anywhere else in the solution. How exactly do you want me to do this?

Reply:
It's called `useDelay`.